src/components/Personalization/in-app-message-actions/actions/displayIframeContent.js

```diff
@@ -150,12 +212,18 @@ const displayHTMLContentInIframe = (settings, collect) => {
 
   const iframe = createIframe(
     content,
-    createIframeClickHandler(container, collect)
+    createIframeClickHandler(container, collect, mobileParameters)
   );
 
   container.appendChild(iframe);
 
-  document.body.append(container);
+  document.body.appendChild(container);
+
+  if (mobileParameters.uiTakeover) {
+    const overlay = createOverlayElement(mobileParameters);
```

Comment:
It seems to me that the overlay element should be added to the DOM before the message `container`.  Then you can remove the zIndex style.  zIndex may not always work, depending on the customers's CSS.  So it's best if we can add elements to the dom in an order that doesn't require it.

Reply:
Done.